.travis.yml

```diff
@@ -22,13 +22,13 @@ before_script:
 
   # macOS dependencies
   - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update; fi
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew tap robotology/cask; fi
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install --only-dependencies yarp; fi
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install gsl sdl sdl_gfx sdl_image homebrew/science/ipopt; fi
   # Fix for opencv and numpy already installed by system
   - if [ "$TRAVIS_OS_NAME" = osx ]; then rm '/usr/local/bin/f2py'; fi
   - if [ "$TRAVIS_OS_NAME" = osx ]; then rm -r '/usr/local/lib/python2.7/site-packages/numpy'; fi
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install opencv; fi
+  # brew dependencies
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew tap robotology/cask; fi
```

Comment:
This should be `robotology/formulae` now 😉

Reply:
You are right... And, again, I was sure I modified this last IFI day... 😲